db/re/skill_nocast_db.txt

```diff
@@ -5,7 +5,7 @@
 // SkillID,Flag
 //
 // Legend for 'Flag' field (bitmask):
-// 1    - cannot be used in normal maps
+// 1    - cannot be used in normal maps (enabled on pvp, gvg, gvg_dungeon, gvg_castle, gvg_te, gvg_te_castle, battleground)
```

Comment:
Just removed the comment. Not really worth the future maintenance/merge conflicts. Was just nice to know for me.